### ISSUE TITLE
Pools containing lanes cannot be resized verticaly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [luisvt/oryx-editor#24]
```
What steps will reproduce the problem?
1. create a pool ( try to resize it ;) )
2. create a lane within the pool
3. resize fails

What is the expected output? What do you see instead?
Pool as well as lane should be resizable in both directions. The pool is not 
(verticaly)!
However, the lane itself is resizable regardless of the pool's bounds


```

Original issue reported on code.google.com by `stefan.k...@hpi-alumni.de` on 26 Nov 2007 at 11:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `stefan.k...@hpi-alumni.de` on 26 Nov 2007 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
we have to think about how resizing a shape object with containing shape 
objects will
manipulate the cotaining shapes' position and size. A solution would also solve 
this
issue.
```

Original comment by `NicoPete...@gmail.com` on 16 Apr 2008 at 3:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `NicoPete...@gmail.com` on 16 Apr 2008 at 4:35
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
will be fixed via issue #70.
```

Original comment by `NicoPete...@gmail.com` on 22 Apr 2008 at 2:24
- Changed state: **Duplicate**
